app/javascript/onboarding/components/ProfileForm.jsx

```diff
@@ -68,7 +69,11 @@ class ProfileForm extends Component {
           </header>
           <div className="current-user-info">
             <figure className="current-user-avatar-container">
-              <img className="current-user-avatar" alt="profile" src={profile_image_90} />
+              <img
+                className="current-user-avatar"
+                alt="profile"
+                src={profile_image_90}
```

Comment:
Doing the following will remove the need for disabling camel case completely in the file.

```suggestion
                src={profile_image_90  /* eslint-disable-line camelcase */}
```

Reply:
So I tried doing this locally before committing the change...but it feels like it clutters up the file a lot more when I disable linting on a line-by-line basis. The JSX and functions feel hard to read with so many `eslint-disable-line`s everywhere instead of just disabling and enabling just once 🤷

Reply:
Ahh, I thought it was only in a couple of places. In that case, leave as is.